man/systemd.time.xml

```diff
@@ -108,37 +108,52 @@
+    thus, in CET, the following are all identical:
+    <literal>Fri 2012-11-23 23:02:15 CET</literal>,
+    <literal>Fri 2012-11-23T23:02:15</literal>,
+    <literal>2012-11-23T23:02:15 CET</literal>,
+    <literal>2012-11-23 23:02:15</literal>.</para>
+
+    <para>The timezone defaults to the current timezone if not specified explicitly.
+    It may be given after a space, like above, in which case it can be:
+    <literal>UTC</literal>,
+    an entry in the installed IANA timezone database (<literal>CET</literal>, <literal>Asia/Tokyo</literal>, &amp;c.;
+    complete list obtainable with <literal>timedatectl
     list-timezones</literal> (see
-    <citerefentry><refentrytitle>timedatectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>).  Using
-    IANA format is recommended over local timezone names, as less prone to errors (e.g. with local timezone
-    it's possible to specify daylight saving time in winter, even though that is not correct). The weekday
-    specification is optional, but when the weekday is specified, it must either be in the abbreviated
+    <citerefentry><refentrytitle>timedatectl</refentrytitle><manvolnum>1</manvolnum></citerefentry>)),
+    or <literal>±<replaceable>05</replaceable></literal>,
+    <literal>±<replaceable>05</replaceable><replaceable>30</replaceable></literal>,
+    <literal>±<replaceable>05</replaceable>:<replaceable>30</replaceable></literal>,
+    <literal>Z</literal>.</para>   <!-- glibc %z -->
+
+    <para>It may also be affixed directly to the timestamp, in which case it must correspond
+    to one of the formats defined in the
+    <ulink url="https://tools.ietf.org/html/rfc3339">RFC 3339</ulink> profile of ISO 8601:
+    <literal>±<replaceable>05</replaceable>:<replaceable>30</replaceable></literal> or <literal>Z</literal>.
+    Thus, the following are also identical to the above:
+    <literal>2012-11-23T23:02:15+01:00</literal>,
+    <literal>2012-11-23 22:02:15Z</literal>.</para>
+
+    <para>A timestamp can start with a field containing a weekday, which can be in an abbreviated
     (<literal>Wed</literal>) or non-abbreviated (<literal>Wednesday</literal>) English language form (case
-    does not matter), and is not subject to the locale choice of the user. Either the date, or the time part
-    may be omitted, in which case the current date or 00:00:00, respectively, is assumed. The seconds
-    component of the time may also be omitted, in which case ":00" is assumed. Year numbers may be specified
-    in full or may be abbreviated (omitting the century).</para>
+    does not matter), regardless of to the locale.
+    However, if a weekday <emphasis>is</emphasis> specified and doesn't match the date, the timestamp is rejected.</para>
 
-    <para>A timestamp is considered invalid if a weekday is specified and the date does not match the specified day of
-    the week.</para>
+    <para>If the date is omitted, it defaults to today. If the time is omitted, it defaults to midnight.
```

Comment:
i think this is not really an improvement. "midnight" at least in my mind means 24:00, and not 00:00. Which is quite a difference, given that we take it relative to a date. This should be explicitly specified as 00:00:00, as before.

Reply:
sure, I hadn't considered someone might read midnight as "end of day" instead of "start of day". cultural I s'pose. reverted to 00:00:00